sdk/messaging/azservicebus/admin/admin_client_queue.go

```diff
@@ -208,31 +178,20 @@ func (ac *Client) GetQueue(ctx context.Context, queueName string, options *GetQu
 		return nil, err
 	}
 
-	props, err := newQueueProperties(&atomResp.Content.QueueDescription)
+	queueItem, err := newQueueItem(atomResp)
 
 	if err != nil {
 		return nil, err
 	}
 
 	return &GetQueueResponse{
-		RawResponse: resp,
-		GetQueueResult: GetQueueResult{
-			QueueProperties: *props,
-		},
+		QueueProperties: queueItem.QueueProperties,
 	}, nil
 }
 
-// GetQueueRuntimePropertiesResult contains the result for Client.GetQueueRuntimeProperties
-type GetQueueRuntimePropertiesResult struct {
```

Comment:
Changelog entry for removing all the Result structs

Reply:
Oof, good point. I swore I had it in there but I DO NOT.

Reply:
Added.